src/main.cpp

```diff
@@ -472,6 +472,10 @@ int main(int argc, char **argv)
 		#endif  // SIGABRT_COMPAT
 	#endif
 
+
+	#ifdef __unix__
+		std::signal(SIGPIPE, SIG_IGN);
```

Comment:
Right, but its action is set to ignore in NonCtrl. For consistency, would you want to do the same for SIGPIPE?